ui/ui-app/src/main/resources/static/locales/en.json

```diff
@@ -814,9 +814,7 @@
     "Monitors All Service Health":"Monitors All Service Health",
     "Total":"Total",
 
-    "service-component-detail.time-locale": "{{date|date:'MM/dd/yyyy'}}",
-    "service-component-detail.time-locale2": "{{date|date:'h:mm'}}",
-    "service-component-detail.at": "at",
+    "service-component-detail.time-locale": "{{date|date:'dd/MM/yyyy @ H:mm'}}",
```

Comment:
Why was this changed from `MM/dd/yyyy 'at' h:mm` to `dd/MM/yyyy @ H:mm`?

Reply:
Because 'a' is used as AM\PM marker in the format string and I can't find the way to change this.
You can change the code on your own if you know how fix this issue.